### ISSUE TITLE
Template requires benchopt >= 1.5.0

### ISSUE THREAD

**Issue by cohenjer** [benchopt/template_benchmark#29]
The current stable benchopt release is 1.4.0, but the template benchmark requires 1.5

This means the following simple instructions do not work:

```bash
   pip install -U benchopt
   git clone git@github.com:benchopt/template_benchmark
   benchopt run template_benchmark
```

This is annoying for writing tutorials, where the simpler the API the better :)

**Comment by tomMoral** [comment]
We should do a release then., we will aim to do it by the end of the week.

**Comment by cohenjer** [comment]
Actually no version of benchopt is called 1.5 even in dev, so technically it is impossible right now to run the template benchmark? This is due to the change in 733f0c5653381e39ee2c0b9f562a259539b90905 ?

**Comment by tomMoral** [comment]
Actually, the min version is `1.4.1` but as we plan to move up a version, we put `1.5` everywhere.
I don't know how to manage `setuptool_scm` to ask that the default dev version increase the minor instead of the bf version.

**Comment by tomMoral** [comment]
This was fixed by #27
